src/components/icons/assistant.tsx

```diff
@@ -1,10 +1,16 @@
+import { ReactSVG, SVGProps } from 'react';
+
 interface AssistantIconProps {
 	size?: number;
 }
 
-export function AssistantIcon( { size = 14 }: AssistantIconProps ) {
+export function AssistantIcon( {
+	size = 14,
+	'aria-hidden': ariaHidden,
+}: AssistantIconProps & SVGProps< ReactSVG > ) {
 	return (
 		<svg
+			aria-hidden={ ariaHidden }
```

Comment:
Previously, the existing `aria-hidden` value introduced in https://github.com/Automattic/studio/pull/202 was not passed down to the child `svg` element.